projects/example-app/src/app/auth/containers/login-page.component.spec.ts

```diff
@@ -18,6 +19,7 @@ describe('Login Page', () => {
       imports: [NoopAnimationsModule, MaterialModule, ReactiveFormsModule],
       declarations: [LoginPageComponent, LoginFormComponent],
       providers: [
+        LoginPageStore,
```

Comment:
issue: cover `OnStateInit` and `OnStoreInit` hooks with tests

Reply:
Agreed. This was for testing example code